speeduino/storage.cpp

```diff
@@ -28,7 +28,7 @@ A full copy of the license may be found in the projects root directory
 #define EEPROM_LAST_BARO              (EEPROM_CALIBRATION_O2_BINS-1)
 
 
-uint32_t deferEEPROMWritesUntil = 0;
+uint32_t lastEEPROMDeferTime;
```

Comment:
Maybe we could use here
 `unsigned long lastEEPROMDeferTime;` 
 or even something like:  `decltype(micros()) lastEEPROMDeferTime;`
 just to make more sure that this variable is same type as `micros()` return value, because when some platforms may have different size the overflow math may not work. Not a critical issue, but just rather for general caution (When some platform have for example 64bit micros() counter the overflow time is anyway too long and it does not matter any more.). I just discovered the `decltype` ,seems to be very convenient.